test/webapi/ows/stac/stac-item.json

```diff
@@ -0,0 +1,1150 @@
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/datacube/v2.1.0/schema.json"
+  ],
+  "type": "Feature",
+  "id": "demo-1w",
+  "bbox": [
+    0.0,
+    50.0,
+    5.0,
+    52.5
+  ],
+  "geometry": {
+    "type": "Polygon",
+    "coordinates": [
+      [
+        [
+          0.0,
+          50.0
+        ],
+        [
+          0.0,
+          52.5
+        ],
+        [
+          5.0,
+          52.5
+        ],
+        [
+          5.0,
+          50.0
+        ],
+        [
+          0.0,
+          50.0
+        ]
+      ]
+    ]
+  },
+  "properties": {
+    "datacube:dimensions": {
```

Comment:
Datacube schema specifies `cube:`, not `datacube:` -- typo?

Reply:
Yes. Stupidity.